stat/distuv/gamma_test.go

```diff
@@ -64,4 +65,11 @@ func testGamma(t *testing.T, f Gamma, i int) {
 	checkExKurtosis(t, i, x, f, 2e-1)
 	checkProbContinuous(t, i, x, f, 1e-3)
 	checkQuantileCDFSurvival(t, i, x, f, 5e-2)
+	if f.Alpha >= 1 {
+		checkMode(t, i, x, f, 2e-1, 1)
+	} else {
+		if !math.IsNaN(f.Mode()) {
+			t.Errorf("Expected NaN mode for alpha < 1, got %v", f.Mode())
+		}
+	}
```

Comment:
Nit. Bad things happen first if possible. (This also makes the condition obviously the same as the message).

```
	if f.Alpha < 1 {
		if !math.IsNaN(f.Mode()) {
			t.Errorf("Expected NaN mode for alpha < 1, got %v", f.Mode())
		}
	} else {
		checkMode(t, i, x, f, 2e-1, 1)
	}
```

Reply:
Done.